posts/2021-04-19-JAXRS-multipart-payloads-21005-beta.adoc

```diff
@@ -0,0 +1,168 @@
+attachments.add(AttachmentBuilder.newBuilder("blogPost")
+                                 .inputStream(new FileInputStream("/path/to/yesterdaysBlogPost.xml"))
+                                 .fileName("myRenamedBlogPost.asciidoc")
+                                 .contentType("text/asciidoc")
+                                 .contentId("myBlogPostID")
+                                 .header("X-PriorityLevel", "Medium")
+                                 .build());
+
+attachments.add(AttachmentBuilder.newBuilder("file1")
+                                 .inputStream("some.xml", new FileInputStream("/path/to/myPicture.png"))
+                                 .contentType("image/png")
+                                 .build());
+
+attachments.add(AttachmentBuilder.newBuilder("authorName")
+                                 .inputStream(new ByteArrayInputStream("John Doe".getBytes()))
+                                 .build());
+
+Response response = client.target(BLOG_SITE_URI)
+                          .request()
+                          .post(Entity.entity(attachments, MediaType.MULTIPART_FORM_DATA));
+----
+
+For more information vist:
+* link:https://tools.ietf.org/html/rfc7578[multipart/form-data RFC 7578]
+* link:JAVADOC[JavaDoc]
+
+[#jee9support]
+=== Jakarta EE 9 Support for oauth-2.0 and samlWeb-2.0
+
+The `oauth-2.0` and `samlWeb-2.0` features have been updated to support Jakarta EE 9. Now these features will automatically adapt to the level of Java EE or Jakarta EE that is already in use. This means that there is no change needed when using them with Jakarta EE 9.
+
+=== Try it now 
+
+To try out these features, just update your build tools to pull the Open Liberty All Beta Features package instead of the main release. The beta works with Java SE 15, Java SE 11, or Java SE 8.
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+  <groupId>io.openliberty.beta</groupId>
+  <artifactId>openliberty-runtime</artifactId>
+  <version>20.0.0.5-beta</version>
+  <type>pom</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty.beta', name: 'openliberty-runtime', version: '[20.0.0.5-beta,)'
+}
+----
+
+Or take a look at our link:{url-prefix}/downloads/#runtime_betas[Downloads page].
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+As of the 21.0.0.2-beta release, Open Liberty is the first vendor product to be Jakarta EE Web Profile 9.0 compatible. With the recent 21.0.0.3-beta release, Open Liberty is the first vendor product to be added to the link:https://jakarta.ee/compatibility/#tab-9[Jakarta EE Platform 9.0 compatibility list].
+
+or more information about Open Liberty's Jakarta EE 9 compatability, view our link:{url-prefix}/blog/2021/03/05/jakarta-ee-9-compatibility.html[Open Liberty beta is Jakarta EE 9 compatible] blog release.
```

Comment:
This Open Liberty beta release comes complete with all of the previously released Jakarta EE9 API candidates. 
 **For** more information...

Reply:
Done